src/applications/personalization/dashboard/components/benefit-application-drafts/SubmissionCard.jsx

```diff
@@ -73,7 +73,7 @@ const SubmissionCard = ({
   const content = (
     <>
       <div className="vads-u-width--full">
-        <h3 aria-describedby="21-0845" className="vads-u-margin-top--0">
+        <h3 aria-describedby="21-0845" className="vads-u-margin-y--0">
```

Comment:
unrelated, but I think this `aria-describedby` value should be `{formId}`

Reply:
actually, we should remove this `aria-describedby` attr.  The same issue exists in the `DraftCard` as well.

Pretty sure this is copy-pasta from before we added the Tag inside of the `h2` for these Cards... likely my bad :D

Reply:
good catch, i completely missed that too hehe